tools/memcached/.dir_placeholder

```diff
@@ -1 +1 @@
-This dir must pre-exist and be owned by the user you are launching awx dev env as. If the dir does not exist before launching the awx dev environment then docker will create the dir and it will be owned by root. Since we start our awx dev environment with user: ${CURRENT_UID} the memcached container will be unable to create a socket file in a directory owned by root.
+This dir must pre-exist and be owned by the user you are launching awx dev env as. If the dir does not exist before launching the awx dev environment then docker will create the dir and it will be owned by root. Since we start our awx dev environment with user: ${CURRENT_UID} Redis will be unable to create a socket file in a directory owned by root.
```

Comment:
Actually, I think we can just entirely `git rm tools/memcached` cc @chrismeyersfsu